src/StockportWebapp/Views/stockportgov/Topic/Index2023.cshtml

```diff
@@ -34,6 +31,10 @@
         </div>
     </section>
 
+    <div class="topic-alerts">
+        <partial name="GlobalAlerts" model='topic.Alerts' />
+    </div>
+
```

Comment:
Could you remove the div around the partial and add the margin-top to the primary items section?